backend/resolvers/Type/Workspace.js

```diff
@@ -1,9 +1,19 @@
+const { checkAccess } = require('../../accessControl')
+
 module.exports = {
   Workspace: {
-    owner(root, args, context) {
+    async tokens(root, args, context) {
+      await checkAccess(context, {
+        checkPrivilege: { requiredPrivilege: 'OWNER', workspaceId: root.id }
+      })
+      return await context.prisma.workspace({
+        id: root.id
+      }).tokens()
+    },
+    participants(root, args, context) {
       return context.prisma.workspace({
         id: root.id
-      }).owner()
+      }).participants()
     },
     defaultCourse(root, args, context) {
```

Comment:
This should probably be removed...

Reply:
I think the default course is still used. Made a new issue for that: #149

Reply:
Will be fixed in upcoming pull-request #150